[INFO][0][<module>] Linux hpc-21-14.cm.cluster 5.14.0-362.24.1.el9_3.x86_64 #1 SMP PREEMPT_DYNAMIC Thu Feb 15 07:18:13 EST 2024 x86_64 x86_64
[INFO][0][<module>] /home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB (2.7)
[INFO][0][<module>] /home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/numpy (1.26.4)
[INFO][0][<module>] /home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/matplotlib (3.9.2)
[INFO][0][<module>] /home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/netCDF4 (1.7.1)
[INFO][0][<module>] /home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/cf_units (3.2.0)
[INFO][0][<module>] /home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/sympy (1.13.2)
[INFO][0][<module>] /home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/mpi4py (4.0.0)
[INFO][0][<module>] 2024-08-29 21:20:19.240640
[DEBUG][0][__init__][CarbonDioxide/HIPPOAToM] 

I am looking for data for the HIPPOAToM confrontation here

/home/renatob/ILAMB_sample/DATA/co2/HIPPOATom/HIPPO_AToM.nc

but I cannot find it. Did you download the data? Have you set the ILAMB_ROOT envronment variable?

[DEBUG][4][__init__][CarbonDioxide/HIPPOAToM] 

I am looking for data for the HIPPOAToM confrontation here

/home/renatob/ILAMB_sample/DATA/co2/HIPPOATom/HIPPO_AToM.nc

but I cannot find it. Did you download the data? Have you set the ILAMB_ROOT envronment variable?

[DEBUG][3][__init__][CarbonDioxide/HIPPOAToM] 

I am looking for data for the HIPPOAToM confrontation here

/home/renatob/ILAMB_sample/DATA/co2/HIPPOATom/HIPPO_AToM.nc

but I cannot find it. Did you download the data? Have you set the ILAMB_ROOT envronment variable?

[DEBUG][13][__init__][CarbonDioxide/HIPPOAToM] 

I am looking for data for the HIPPOAToM confrontation here

/home/renatob/ILAMB_sample/DATA/co2/HIPPOATom/HIPPO_AToM.nc

but I cannot find it. Did you download the data? Have you set the ILAMB_ROOT envronment variable?

[DEBUG][12][__init__][CarbonDioxide/HIPPOAToM] 

I am looking for data for the HIPPOAToM confrontation here

/home/renatob/ILAMB_sample/DATA/co2/HIPPOATom/HIPPO_AToM.nc

but I cannot find it. Did you download the data? Have you set the ILAMB_ROOT envronment variable?

[DEBUG][1][__init__][CarbonDioxide/HIPPOAToM] 

I am looking for data for the HIPPOAToM confrontation here

/home/renatob/ILAMB_sample/DATA/co2/HIPPOATom/HIPPO_AToM.nc

but I cannot find it. Did you download the data? Have you set the ILAMB_ROOT envronment variable?

[DEBUG][2][__init__][CarbonDioxide/HIPPOAToM] 

I am looking for data for the HIPPOAToM confrontation here

/home/renatob/ILAMB_sample/DATA/co2/HIPPOATom/HIPPO_AToM.nc

but I cannot find it. Did you download the data? Have you set the ILAMB_ROOT envronment variable?

[DEBUG][8][__init__][CarbonDioxide/HIPPOAToM] 

I am looking for data for the HIPPOAToM confrontation here

/home/renatob/ILAMB_sample/DATA/co2/HIPPOATom/HIPPO_AToM.nc

but I cannot find it. Did you download the data? Have you set the ILAMB_ROOT envronment variable?

[DEBUG][10][__init__][CarbonDioxide/HIPPOAToM] 

I am looking for data for the HIPPOAToM confrontation here

/home/renatob/ILAMB_sample/DATA/co2/HIPPOATom/HIPPO_AToM.nc

but I cannot find it. Did you download the data? Have you set the ILAMB_ROOT envronment variable?

[DEBUG][6][__init__][CarbonDioxide/HIPPOAToM] 

I am looking for data for the HIPPOAToM confrontation here

/home/renatob/ILAMB_sample/DATA/co2/HIPPOATom/HIPPO_AToM.nc

but I cannot find it. Did you download the data? Have you set the ILAMB_ROOT envronment variable?

[DEBUG][11][__init__][CarbonDioxide/HIPPOAToM] 

I am looking for data for the HIPPOAToM confrontation here

/home/renatob/ILAMB_sample/DATA/co2/HIPPOATom/HIPPO_AToM.nc

but I cannot find it. Did you download the data? Have you set the ILAMB_ROOT envronment variable?

[DEBUG][9][__init__][CarbonDioxide/HIPPOAToM] 

I am looking for data for the HIPPOAToM confrontation here

/home/renatob/ILAMB_sample/DATA/co2/HIPPOATom/HIPPO_AToM.nc

but I cannot find it. Did you download the data? Have you set the ILAMB_ROOT envronment variable?

[DEBUG][7][__init__][CarbonDioxide/HIPPOAToM] 

I am looking for data for the HIPPOAToM confrontation here

/home/renatob/ILAMB_sample/DATA/co2/HIPPOATom/HIPPO_AToM.nc

but I cannot find it. Did you download the data? Have you set the ILAMB_ROOT envronment variable?

[DEBUG][5][__init__][CarbonDioxide/HIPPOAToM] 

I am looking for data for the HIPPOAToM confrontation here

/home/renatob/ILAMB_sample/DATA/co2/HIPPOATom/HIPPO_AToM.nc

but I cannot find it. Did you download the data? Have you set the ILAMB_ROOT envronment variable?

[DEBUG][14][__init__][CarbonDioxide/HIPPOAToM] 

I am looking for data for the HIPPOAToM confrontation here

/home/renatob/ILAMB_sample/DATA/co2/HIPPOATom/HIPPO_AToM.nc

but I cannot find it. Did you download the data? Have you set the ILAMB_ROOT envronment variable?

[DEBUG][15][__init__][CarbonDioxide/HIPPOAToM] 

I am looking for data for the HIPPOAToM confrontation here

/home/renatob/ILAMB_sample/DATA/co2/HIPPOATom/HIPPO_AToM.nc

but I cannot find it. Did you download the data? Have you set the ILAMB_ROOT envronment variable?

[INFO][0][MakeComparable][Biomass/ESACCI][CARDAMOM] Spatial data was clipped from the reference:  before: [  1 281 721] after: [  1 269 711]
[INFO][0][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][0][confront][Biomass/ESACCI][CARDAMOM] Success
[DEBUG][0][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [biomass,cVeg] in the model results
[DEBUG][0][WorkConfront][Biomass/ESACCI][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][0][MakeComparable][Biomass/ESACCI][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [  1 281 721] after: [  1 269 711]
[INFO][0][MakeComparable][Biomass/ESACCI][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [ 12  96 144] after: [ 12  72 142]
[INFO][0][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][0][confront][Biomass/ESACCI][ENSEMBLE-CMIP6] Success
[INFO][0][MakeComparable][Biomass/GEOCARBON][CARDAMOM] Spatial data was clipped from the reference:  before: [  1 281 720] after: [  1 281 710]
[INFO][0][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][0][confront][Biomass/GEOCARBON][CARDAMOM] Success
[INFO][0][MakeComparable][Biomass/GEOCARBON][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [  1 281 720] after: [  1 281 710]
[INFO][0][MakeComparable][Biomass/GEOCARBON][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [ 72  96 144] after: [ 72  73 142]
[INFO][0][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][0][confront][Biomass/GEOCARBON][ENSEMBLE-CMIP5] Success
[INFO][0][MakeComparable][Biomass/GEOCARBON][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [  1 281 720] after: [  1 281 710]
[INFO][0][MakeComparable][Biomass/GEOCARBON][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [120  96 144] after: [120  73 142]
[INFO][0][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][0][confront][Biomass/GEOCARBON][ENSEMBLE-CMIP6] Success
[INFO][0][MakeComparable][Biomass/NBCD2000][CARDAMOM] Spatial data was clipped from the reference:  before: [  1 360 720] after: [  1 273 710]
[INFO][0][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][0][confront][Biomass/NBCD2000][CARDAMOM] Success
[INFO][0][MakeComparable][Biomass/NBCD2000][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [  1 360 720] after: [  1 273 710]
[INFO][0][MakeComparable][Biomass/NBCD2000][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [120  96 144] after: [120  73 142]
[INFO][0][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][0][confront][Biomass/NBCD2000][ENSEMBLE-CMIP5] Success
[INFO][0][MakeComparable][Biomass/NBCD2000][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [  1 360 720] after: [  1 273 710]
[INFO][0][MakeComparable][Biomass/NBCD2000][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [120  96 144] after: [120  73 142]
[INFO][0][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][0][confront][Biomass/NBCD2000][ENSEMBLE-CMIP6] Success
[INFO][0][MakeComparable][Biomass/Saatchi2011][CARDAMOM] Spatial data was clipped from the reference:  before: [  1 360 720] after: [  1 273 710]
[INFO][0][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][0][confront][Biomass/Saatchi2011][CARDAMOM] Success
[INFO][0][MakeComparable][Biomass/Saatchi2011][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [  1 360 720] after: [  1 273 710]
[INFO][0][MakeComparable][Biomass/Saatchi2011][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [132  96 144] after: [132  73 142]
[INFO][0][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][0][confront][Biomass/Saatchi2011][ENSEMBLE-CMIP5] Success
[INFO][0][MakeComparable][Biomass/Saatchi2011][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [  1 360 720] after: [  1 273 710]
[INFO][0][MakeComparable][Biomass/Saatchi2011][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [132  96 144] after: [132  73 142]
[INFO][0][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][0][confront][Biomass/Saatchi2011][ENSEMBLE-CMIP6] Success
[INFO][0][MakeComparable][Biomass/Thurner][CARDAMOM] Spatial data was clipped from the reference:  before: [  1 360 720] after: [  1 273 710]
[INFO][0][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][0][confront][Biomass/Thurner][CARDAMOM] Success
[INFO][0][MakeComparable][Biomass/Thurner][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [  1 360 720] after: [  1 273 710]
[INFO][0][MakeComparable][Biomass/Thurner][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [194  96 144] after: [194  73 142]
[INFO][0][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][0][confront][Biomass/Thurner][ENSEMBLE-CMIP5] Success
[INFO][0][MakeComparable][Biomass/Thurner][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [  1 360 720] after: [  1 273 710]
[INFO][0][MakeComparable][Biomass/Thurner][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [241  96 144] after: [241  73 142]
[INFO][0][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][0][confront][Biomass/Thurner][ENSEMBLE-CMIP6] Success
[INFO][0][MakeComparable][Biomass/USForest][CARDAMOM] Spatial data was clipped from the reference:  before: [  1 360 720] after: [  1 273 710]
[INFO][0][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][0][confront][Biomass/USForest][CARDAMOM] Success
[INFO][0][MakeComparable][Biomass/USForest][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [  1 360 720] after: [  1 273 710]
[INFO][0][MakeComparable][Biomass/USForest][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [120  96 144] after: [120  73 142]
[INFO][0][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][0][confront][Biomass/USForest][ENSEMBLE-CMIP5] Success
[DEBUG][4][extractTimeSeries][CARDAMOM] Could not find [fBNF] in the model results
[DEBUG][4][WorkConfront][NitrogenFixation/Davies-Barnard][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][4][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [fBNF] in the model results
[DEBUG][4][WorkConfront][NitrogenFixation/Davies-Barnard][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][4][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [fBNF] in the model results
[DEBUG][4][WorkConfront][NitrogenFixation/Davies-Barnard][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][1][MakeComparable][Biomass/USForest][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [  1 360 720] after: [  1 273 710]
[INFO][4][AnalysisMeanStateSites][FCH4] Bias scored using Collier2018
[INFO][4][AnalysisMeanStateSites][FCH4] RMSE scored using Collier2018
[INFO][1][MakeComparable][Biomass/USForest][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [120  96 144] after: [120  73 142]
[INFO][4][confront][Methane/FluxnetANN][CARDAMOM] Success
[DEBUG][4][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [FCH4,ch4] in the model results
[DEBUG][4][WorkConfront][Methane/FluxnetANN][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][4][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [FCH4,ch4] in the model results
[DEBUG][4][WorkConfront][Methane/FluxnetANN][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][12][extractTimeSeries][CARDAMOM] Could not find [rlus] in the model results
[DEBUG][12][WorkConfront][SurfaceNetRadiation/WRMC.BSRN][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 409, in extractTimeSeries
    v = self.derivedVariable(
        ^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 495, in derivedVariable
    Vs[arg] = self.extractTimeSeries(
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][12][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [rlus] in the model results
[DEBUG][12][WorkConfront][SurfaceNetRadiation/WRMC.BSRN][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 409, in extractTimeSeries
    v = self.derivedVariable(
        ^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 495, in derivedVariable
    Vs[arg] = self.extractTimeSeries(
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][12][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [rlus] in the model results
[DEBUG][12][WorkConfront][SurfaceNetRadiation/WRMC.BSRN][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 409, in extractTimeSeries
    v = self.derivedVariable(
        ^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 495, in derivedVariable
    Vs[arg] = self.extractTimeSeries(
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][12][extractTimeSeries][CARDAMOM] Could not find [rlus] in the model results
[DEBUG][12][WorkConfront][SurfaceNetRadiation/CLASS][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 409, in extractTimeSeries
    v = self.derivedVariable(
        ^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 495, in derivedVariable
    Vs[arg] = self.extractTimeSeries(
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][2][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [gpp] in the model results
[DEBUG][2][WorkConfront][GrossPrimaryProductivity/WECANN][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][12][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [rlus] in the model results
[DEBUG][12][WorkConfront][SurfaceNetRadiation/CLASS][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 409, in extractTimeSeries
    v = self.derivedVariable(
        ^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 495, in derivedVariable
    Vs[arg] = self.extractTimeSeries(
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][2][MakeComparable][GrossPrimaryProductivity/WECANN][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [108 180 360] after: [108 137 355]
[INFO][2][MakeComparable][GrossPrimaryProductivity/WECANN][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [ 96  96 144] after: [ 96  73 142]
[INFO][4][MakeComparable][Evapotranspiration/GLEAMv3.3a][CARDAMOM] Spatial data was clipped from the reference:  before: [468 360 720] after: [468 273 710]
[DEBUG][12][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [rlus] in the model results
[DEBUG][12][WorkConfront][SurfaceNetRadiation/CLASS][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 409, in extractTimeSeries
    v = self.derivedVariable(
        ^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 495, in derivedVariable
    Vs[arg] = self.extractTimeSeries(
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][3][AnalysisMeanStateSites][nee] Bias scored using Collier2018
[INFO][3][AnalysisMeanStateSites][nee] RMSE scored using Collier2018
[INFO][3][confront][NetEcosystemExchange/FLUXNET2015][ENSEMBLE-CMIP5] Success
[INFO][12][MakeComparable][GroundHeatFlux/CLASS][CARDAMOM] Spatial data was clipped from the reference:  before: [ 84 360 720] after: [ 84 273 710]
[DEBUG][13][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [tasmax] in the model results
[DEBUG][13][WorkConfront][DiurnalMaxTemperature/CRU4.02][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][1][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][1][confront][Biomass/USForest][ENSEMBLE-CMIP6] Success
[INFO][3][AnalysisMeanStateSites][nee] Bias scored using Collier2018
[INFO][3][AnalysisMeanStateSites][nee] RMSE scored using Collier2018
[INFO][3][confront][NetEcosystemExchange/FLUXNET2015][ENSEMBLE-CMIP6] Success
[INFO][1][MakeComparable][Biomass/XuSaatchi2021][CARDAMOM] Spatial data was clipped from the reference:  before: [ 20 360 720] after: [ 20 273 710]
[INFO][3][AnalysisMeanStateSites][reco] Bias scored using Collier2018
[INFO][3][AnalysisMeanStateSites][reco] RMSE scored using Collier2018
[INFO][3][confront][EcosystemRespiration/FLUXNET2015][CARDAMOM] Success
[INFO][3][AnalysisMeanStateSites][reco] Bias scored using Collier2018
[INFO][3][AnalysisMeanStateSites][reco] RMSE scored using Collier2018
[INFO][3][confront][EcosystemRespiration/FLUXNET2015][ENSEMBLE-CMIP5] Success
[INFO][3][AnalysisMeanStateSites][reco] Bias scored using Collier2018
[INFO][3][AnalysisMeanStateSites][reco] RMSE scored using Collier2018
[INFO][3][confront][EcosystemRespiration/FLUXNET2015][ENSEMBLE-CMIP6] Success
[INFO][3][MakeComparable][EcosystemRespiration/FLUXCOM][CARDAMOM] Spatial data was clipped from the reference:  before: [408 360 720] after: [408 273 710]
[INFO][13][MakeComparable][DiurnalMinTemperature/CRU4.02][CARDAMOM] Spatial data was clipped from the reference:  before: [696 360 720] after: [696 273 710]
[DEBUG][13][WorkConfront][DiurnalMinTemperature/CRU4.02][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 380, in stageData
    obs, mod = il.MakeComparable(
               ^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ilamblib.py", line 2570, in MakeComparable
    com = com.convert(ref.unit)
          ^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Variable.py", line 1022, in convert
    src_unit = Unit(self.unit)
               ^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/cf_units/__init__.py", line 830, in __init__
    raise value_error from None
ValueError: [UT_UNKNOWN] Failed to parse unit "deg C"

[DEBUG][13][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [tasmin] in the model results
[DEBUG][13][WorkConfront][DiurnalMinTemperature/CRU4.02][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][1][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][1][confront][Biomass/XuSaatchi2021][CARDAMOM] Success
[INFO][1][MakeComparable][Biomass/XuSaatchi2021][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [ 20 360 720] after: [ 20 273 710]
[INFO][1][MakeComparable][Biomass/XuSaatchi2021][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [ 72  96 144] after: [ 72  73 142]
[INFO][2][AnalysisMeanStateSpace][gpp] Bias scored using Collier2018
[DEBUG][13][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [tasmin] in the model results
[DEBUG][13][WorkConfront][DiurnalMinTemperature/CRU4.02][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][2][AnalysisMeanStateSpace][gpp] RMSE scored using Collier2018
[INFO][2][confront][GrossPrimaryProductivity/WECANN][ENSEMBLE-CMIP6] Success
[DEBUG][13][WorkConfront][DiurnalTemperatureRange/CRU4.02][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 409, in extractTimeSeries
    v = self.derivedVariable(
        ^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 549, in derivedVariable
    result, unit = il.SympifyWithArgsUnits(expression, args, units)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ilamblib.py", line 616, in SympifyWithArgsUnits
    Unit(units[key]).convert(np.ones(1), Unit(units[key0]))
    ^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/cf_units/__init__.py", line 830, in __init__
    raise value_error from None
ValueError: [UT_UNKNOWN] Failed to parse unit "deg C"

[INFO][2][MakeComparable][LeafAreaIndex/AVHRR][CARDAMOM] Spatial data was clipped from the reference:  before: [348 360 720] after: [348 273 710]
[DEBUG][13][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [tasmax] in the model results
[DEBUG][13][WorkConfront][DiurnalTemperatureRange/CRU4.02][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 409, in extractTimeSeries
    v = self.derivedVariable(
        ^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 495, in derivedVariable
    Vs[arg] = self.extractTimeSeries(
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][1][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][1][confront][Biomass/XuSaatchi2021][ENSEMBLE-CMIP5] Success
[INFO][1][MakeComparable][Biomass/XuSaatchi2021][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [ 20 360 720] after: [ 20 273 710]
[INFO][1][MakeComparable][Biomass/XuSaatchi2021][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [180  96 144] after: [180  73 142]
[DEBUG][13][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [tasmax] in the model results
[DEBUG][13][WorkConfront][DiurnalTemperatureRange/CRU4.02][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 409, in extractTimeSeries
    v = self.derivedVariable(
        ^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 495, in derivedVariable
    Vs[arg] = self.extractTimeSeries(
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][12][AnalysisMeanStateSpace][hfdsl] Bias scored using Collier2018
[INFO][13][MakeComparable][Precipitation/CMAPv1904][CARDAMOM] Spatial data was clipped from the reference:  before: [480 360 720] after: [480 273 710]
[INFO][12][AnalysisMeanStateSpace][hfdsl] RMSE scored using Collier2018
[INFO][12][confront][GroundHeatFlux/CLASS][CARDAMOM] Success
[DEBUG][12][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [hfdsl,FGR12] in the model results
[DEBUG][12][WorkConfront][GroundHeatFlux/CLASS][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][12][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [hfdsl,FGR12] in the model results
[DEBUG][12][WorkConfront][GroundHeatFlux/CLASS][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][1][AnalysisMeanStateSpace][biomass] Bias scored using Collier2018
[INFO][1][confront][Biomass/XuSaatchi2021][ENSEMBLE-CMIP6] Success
[DEBUG][1][extractTimeSeries][CARDAMOM] Could not find [burntArea,burntFractionAll] in the model results
[DEBUG][1][WorkConfront][BurnedArea/GFED4.1S][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][3][AnalysisMeanStateSpace][reco] Bias scored using Collier2018
[DEBUG][1][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [burntArea,burntFractionAll] in the model results
[DEBUG][1][WorkConfront][BurnedArea/GFED4.1S][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][12][extractTimeSeries][CARDAMOM] Could not find [tas] in the model results
[DEBUG][12][WorkConfront][SurfaceAirTemperature/CRU4.02][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][1][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [burntArea,burntFractionAll] in the model results
[DEBUG][1][WorkConfront][BurnedArea/GFED4.1S][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][1][extractTimeSeries][CARDAMOM] Could not find [nbp] in the model results
[DEBUG][1][WorkConfront][CarbonDioxide/NOAA.Emulated][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ConfCO2.py", line 602, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ConfCO2.py", line 333, in stageData
    mod = self.emulatedModelResult(m, obs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ConfCO2.py", line 213, in emulatedModelResult
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][3][AnalysisMeanStateSpace][reco] RMSE scored using Collier2018
[INFO][3][confront][EcosystemRespiration/FLUXCOM][CARDAMOM] Success
[INFO][12][MakeComparable][SurfaceAirTemperature/CRU4.02][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [456 360 720] after: [456 273 710]
[INFO][12][MakeComparable][SurfaceAirTemperature/CRU4.02][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [312  96 144] after: [312  73 142]
[INFO][3][MakeComparable][EcosystemRespiration/FLUXCOM][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [408 360 720] after: [408 273 710]
[INFO][3][MakeComparable][EcosystemRespiration/FLUXCOM][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [312  96 144] after: [312  73 142]
[INFO][4][AnalysisMeanStateSpace][et] Bias scored using Collier2018
[INFO][2][AnalysisMeanStateSpace][lai] Bias scored using Collier2018
[INFO][4][AnalysisMeanStateSpace][et] RMSE scored using Collier2018
[INFO][4][confront][Evapotranspiration/GLEAMv3.3a][CARDAMOM] Success
[DEBUG][4][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [et,evspsbl] in the model results
[DEBUG][4][WorkConfront][Evapotranspiration/GLEAMv3.3a][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][2][AnalysisMeanStateSpace][lai] RMSE scored using Collier2018
[INFO][2][confront][LeafAreaIndex/AVHRR][CARDAMOM] Success
[DEBUG][4][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [et,evspsbl] in the model results
[DEBUG][4][WorkConfront][Evapotranspiration/GLEAMv3.3a][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][4][MakeComparable][Evapotranspiration/MODIS][CARDAMOM] Spatial data was clipped from the reference:  before: [168 360 720] after: [168 273 710]
[INFO][2][MakeComparable][LeafAreaIndex/AVHRR][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [348 360 720] after: [348 273 710]
[INFO][2][MakeComparable][LeafAreaIndex/AVHRR][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [288  96 144] after: [288  73 142]
[INFO][13][AnalysisMeanStateSpace][pr] Bias scored using Collier2018
[INFO][13][AnalysisMeanStateSpace][pr] RMSE scored using Collier2018
[INFO][13][confront][Precipitation/CMAPv1904][CARDAMOM] Success
[INFO][13][MakeComparable][Precipitation/CMAPv1904][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [480 360 720] after: [480 273 710]
[INFO][13][MakeComparable][Precipitation/CMAPv1904][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [324  96 144] after: [324  73 142]
[INFO][3][AnalysisMeanStateSpace][reco] Bias scored using Collier2018
[INFO][4][AnalysisMeanStateSpace][et] Bias scored using Collier2018
[INFO][3][AnalysisMeanStateSpace][reco] RMSE scored using Collier2018
[INFO][3][confront][EcosystemRespiration/FLUXCOM][ENSEMBLE-CMIP5] Success
[INFO][4][AnalysisMeanStateSpace][et] RMSE scored using Collier2018
[INFO][4][confront][Evapotranspiration/MODIS][CARDAMOM] Success
[DEBUG][4][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [et,evspsbl] in the model results
[DEBUG][4][WorkConfront][Evapotranspiration/MODIS][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][3][MakeComparable][EcosystemRespiration/FLUXCOM][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [408 360 720] after: [408 273 710]
[INFO][3][MakeComparable][EcosystemRespiration/FLUXCOM][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [408  96 144] after: [408  73 142]
[DEBUG][4][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [et,evspsbl] in the model results
[DEBUG][4][WorkConfront][Evapotranspiration/MODIS][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][4][MakeComparable][Evapotranspiration/MOD16A2][CARDAMOM] Spatial data was clipped from the reference:  before: [180 360 720] after: [180 273 710]
[INFO][12][AnalysisMeanStateSpace][tas] Bias scored using Collier2018
[INFO][12][AnalysisMeanStateSpace][tas] RMSE scored using Collier2018
[INFO][12][confront][SurfaceAirTemperature/CRU4.02][ENSEMBLE-CMIP5] Success
[INFO][12][MakeComparable][SurfaceAirTemperature/CRU4.02][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [456 360 720] after: [456 273 710]
[INFO][12][MakeComparable][SurfaceAirTemperature/CRU4.02][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [420  96 144] after: [420  73 142]
[INFO][2][AnalysisMeanStateSpace][lai] Bias scored using Collier2018
[INFO][2][AnalysisMeanStateSpace][lai] RMSE scored using Collier2018
[INFO][2][confront][LeafAreaIndex/AVHRR][ENSEMBLE-CMIP5] Success
[INFO][2][MakeComparable][LeafAreaIndex/AVHRR][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [348 360 720] after: [348 273 710]
[INFO][2][MakeComparable][LeafAreaIndex/AVHRR][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [348  96 144] after: [348  73 142]
[INFO][4][AnalysisMeanStateSpace][et] Bias scored using Collier2018
[INFO][4][AnalysisMeanStateSpace][et] RMSE scored using Collier2018
[INFO][4][confront][Evapotranspiration/MOD16A2][CARDAMOM] Success
[INFO][13][AnalysisMeanStateSpace][pr] Bias scored using Collier2018
[DEBUG][4][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [et,evspsbl] in the model results
[DEBUG][4][WorkConfront][Evapotranspiration/MOD16A2][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][4][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [et,evspsbl] in the model results
[DEBUG][4][WorkConfront][Evapotranspiration/MOD16A2][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][13][AnalysisMeanStateSpace][pr] RMSE scored using Collier2018
[INFO][13][confront][Precipitation/CMAPv1904][ENSEMBLE-CMIP5] Success
[DEBUG][4][extractTimeSeries][CARDAMOM] Could not find [hfss,FSH] in the model results
[DEBUG][4][WorkConfront][EvaporativeFraction/FLUXCOM][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ConfEvapFraction.py", line 168, in confront
    obs, mod, obs_timeint, mod_timeint = self.stageData(m)
                                         ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ConfEvapFraction.py", line 73, in stageData
    sh_mod = m.extractTimeSeries(
             ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][13][MakeComparable][Precipitation/CMAPv1904][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [480 360 720] after: [480 273 710]
[INFO][13][MakeComparable][Precipitation/CMAPv1904][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [432  96 144] after: [432  73 142]
[DEBUG][4][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [hfss,FSH] in the model results
[DEBUG][4][WorkConfront][EvaporativeFraction/FLUXCOM][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ConfEvapFraction.py", line 168, in confront
    obs, mod, obs_timeint, mod_timeint = self.stageData(m)
                                         ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ConfEvapFraction.py", line 73, in stageData
    sh_mod = m.extractTimeSeries(
             ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][2][AnalysisMeanStateSpace][lai] Bias scored using Collier2018
[INFO][2][AnalysisMeanStateSpace][lai] RMSE scored using Collier2018
[INFO][2][confront][LeafAreaIndex/AVHRR][ENSEMBLE-CMIP6] Success
[INFO][2][MakeComparable][LeafAreaIndex/AVH15C1][CARDAMOM] Spatial data was clipped from the reference:  before: [468 360 720] after: [468 273 710]
[INFO][12][AnalysisMeanStateSpace][tas] Bias scored using Collier2018
[INFO][12][AnalysisMeanStateSpace][tas] RMSE scored using Collier2018
[INFO][12][confront][SurfaceAirTemperature/CRU4.02][ENSEMBLE-CMIP6] Success
[DEBUG][12][extractTimeSeries][CARDAMOM] Could not find [tas] in the model results
[DEBUG][12][WorkConfront][SurfaceAirTemperature/FLUXNET2015][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][12][AnalysisMeanStateSites][tas] Bias scored using Collier2018
[INFO][12][AnalysisMeanStateSites][tas] RMSE scored using Collier2018
[INFO][12][confront][SurfaceAirTemperature/FLUXNET2015][ENSEMBLE-CMIP5] Success
[INFO][12][AnalysisMeanStateSites][tas] Bias scored using Collier2018
[INFO][12][AnalysisMeanStateSites][tas] RMSE scored using Collier2018
[INFO][12][confront][SurfaceAirTemperature/FLUXNET2015][ENSEMBLE-CMIP6] Success
[INFO][12][MakeComparable][DiurnalMaxTemperature/CRU4.02][CARDAMOM] Spatial data was clipped from the reference:  before: [696 360 720] after: [696 273 710]
[DEBUG][12][WorkConfront][DiurnalMaxTemperature/CRU4.02][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 380, in stageData
    obs, mod = il.MakeComparable(
               ^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ilamblib.py", line 2570, in MakeComparable
    com = com.convert(ref.unit)
          ^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Variable.py", line 1022, in convert
    src_unit = Unit(self.unit)
               ^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/cf_units/__init__.py", line 830, in __init__
    raise value_error from None
ValueError: [UT_UNKNOWN] Failed to parse unit "deg C"

[DEBUG][12][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [tasmax] in the model results
[DEBUG][12][WorkConfront][DiurnalMaxTemperature/CRU4.02][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][15][MakeComparable][SurfaceDownwardSWRadiation/GEWEX.SRB][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [288 360 720] after: [288 273 710]
[INFO][15][MakeComparable][SurfaceDownwardSWRadiation/GEWEX.SRB][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [264  96 144] after: [264  73 142]
[INFO][14][MakeComparable][Precipitation/GPCPv2.3][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [480 360 720] after: [480 273 710]
[INFO][14][MakeComparable][Precipitation/GPCPv2.3][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [432  96 144] after: [432  73 142]
[INFO][2][AnalysisMeanStateSpace][lai] Bias scored using Collier2018
[INFO][2][AnalysisMeanStateSpace][lai] RMSE scored using Collier2018
[INFO][2][confront][LeafAreaIndex/AVH15C1][CARDAMOM] Success
[INFO][13][AnalysisMeanStateSpace][pr] Bias scored using Collier2018
[INFO][2][MakeComparable][LeafAreaIndex/AVH15C1][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [468 360 720] after: [468 273 710]
[INFO][2][MakeComparable][LeafAreaIndex/AVH15C1][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [300  96 144] after: [300  73 142]
[INFO][13][AnalysisMeanStateSpace][pr] RMSE scored using Collier2018
[INFO][13][confront][Precipitation/CMAPv1904][ENSEMBLE-CMIP6] Success
[INFO][13][AnalysisMeanStateSites][pr] Bias scored using Collier2018
[INFO][13][AnalysisMeanStateSites][pr] RMSE scored using Collier2018
[INFO][13][confront][Precipitation/FLUXNET2015][CARDAMOM] Success
[INFO][13][AnalysisMeanStateSites][pr] Bias scored using Collier2018
[INFO][13][AnalysisMeanStateSites][pr] RMSE scored using Collier2018
[INFO][13][confront][Precipitation/FLUXNET2015][ENSEMBLE-CMIP5] Success
[INFO][13][AnalysisMeanStateSites][pr] Bias scored using Collier2018
[INFO][13][AnalysisMeanStateSites][pr] RMSE scored using Collier2018
[INFO][13][confront][Precipitation/FLUXNET2015][ENSEMBLE-CMIP6] Success
[INFO][13][MakeComparable][Precipitation/GPCCv2018][CARDAMOM] Spatial data was clipped from the reference:  before: [444 360 720] after: [444 273 710]
[INFO][15][AnalysisMeanStateSpace][rsds] Bias scored using Collier2018
[INFO][15][AnalysisMeanStateSpace][rsds] RMSE scored using Collier2018
[INFO][15][confront][SurfaceDownwardSWRadiation/GEWEX.SRB][ENSEMBLE-CMIP5] Success
[INFO][13][AnalysisMeanStateSpace][pr] Bias scored using Collier2018
[INFO][15][MakeComparable][SurfaceDownwardSWRadiation/GEWEX.SRB][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [288 360 720] after: [288 273 710]
[INFO][15][MakeComparable][SurfaceDownwardSWRadiation/GEWEX.SRB][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [288  96 144] after: [288  73 142]
[INFO][13][AnalysisMeanStateSpace][pr] RMSE scored using Collier2018
[INFO][13][confront][Precipitation/GPCCv2018][CARDAMOM] Success
[INFO][13][MakeComparable][Precipitation/GPCCv2018][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [444 360 720] after: [444 273 710]
[INFO][13][MakeComparable][Precipitation/GPCCv2018][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [312  96 144] after: [312  73 142]
[INFO][2][AnalysisMeanStateSpace][lai] Bias scored using Collier2018
[INFO][2][AnalysisMeanStateSpace][lai] RMSE scored using Collier2018
[INFO][2][confront][LeafAreaIndex/AVH15C1][ENSEMBLE-CMIP5] Success
[INFO][2][MakeComparable][LeafAreaIndex/AVH15C1][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [468 360 720] after: [468 273 710]
[INFO][2][MakeComparable][LeafAreaIndex/AVH15C1][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [408  96 144] after: [408  73 142]
[INFO][14][AnalysisMeanStateSpace][pr] Bias scored using Collier2018
[INFO][14][AnalysisMeanStateSpace][pr] RMSE scored using Collier2018
[INFO][14][confront][Precipitation/GPCPv2.3][ENSEMBLE-CMIP6] Success
[INFO][14][MakeComparable][Precipitation/CLASS][CARDAMOM] Spatial data was clipped from the reference:  before: [ 84 360 720] after: [ 84 273 710]
[DEBUG][14][WorkConfront][Precipitation/CLASS][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 464, in confront
    il.AnalysisMeanStateSpace(
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ilamblib.py", line 1751, in AnalysisMeanStateSpace
    REF = ref.interpolate(lat=lat, lon=lon, lat_bnds=lat_bnds, lon_bnds=lon_bnds)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Variable.py", line 1544, in interpolate
    data_bnds = self.data_bnds[ind]
                ~~~~~~~~~~~~~~^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/numpy/ma/core.py", line 3228, in __getitem__
    dout = self.data[indx]
           ~~~~~~~~~^^^^^^
IndexError: too many indices for array: array is 3-dimensional, but 4 were indexed

[INFO][14][MakeComparable][Precipitation/CLASS][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [ 84 360 720] after: [ 84 273 710]
[INFO][14][MakeComparable][Precipitation/CLASS][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [ 36  96 144] after: [ 36  73 142]
[DEBUG][14][WorkConfront][Precipitation/CLASS][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 464, in confront
    il.AnalysisMeanStateSpace(
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ilamblib.py", line 1751, in AnalysisMeanStateSpace
    REF = ref.interpolate(lat=lat, lon=lon, lat_bnds=lat_bnds, lon_bnds=lon_bnds)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Variable.py", line 1544, in interpolate
    data_bnds = self.data_bnds[ind]
                ~~~~~~~~~~~~~~^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/numpy/ma/core.py", line 3228, in __getitem__
    dout = self.data[indx]
           ~~~~~~~~~^^^^^^
IndexError: too many indices for array: array is 3-dimensional, but 4 were indexed

[INFO][14][MakeComparable][Precipitation/CLASS][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [ 84 360 720] after: [ 84 273 710]
[INFO][14][MakeComparable][Precipitation/CLASS][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [ 84  96 144] after: [ 84  73 142]
[DEBUG][14][WorkConfront][Precipitation/CLASS][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 464, in confront
    il.AnalysisMeanStateSpace(
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ilamblib.py", line 1751, in AnalysisMeanStateSpace
    REF = ref.interpolate(lat=lat, lon=lon, lat_bnds=lat_bnds, lon_bnds=lon_bnds)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Variable.py", line 1544, in interpolate
    data_bnds = self.data_bnds[ind]
                ~~~~~~~~~~~~~~^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/numpy/ma/core.py", line 3228, in __getitem__
    dout = self.data[indx]
           ~~~~~~~~~^^^^^^
IndexError: too many indices for array: array is 3-dimensional, but 4 were indexed

[DEBUG][14][extractTimeSeries][CARDAMOM] Could not find [rhums,hurs] in the model results
[DEBUG][14][WorkConfront][SurfaceRelativeHumidity/ERA5][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][15][AnalysisMeanStateSpace][rsds] Bias scored using Collier2018
[INFO][13][AnalysisMeanStateSpace][pr] Bias scored using Collier2018
[DEBUG][14][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [rhums,hurs] in the model results
[DEBUG][14][WorkConfront][SurfaceRelativeHumidity/ERA5][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][14][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [rhums,hurs] in the model results
[DEBUG][14][WorkConfront][SurfaceRelativeHumidity/ERA5][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][15][AnalysisMeanStateSpace][rsds] RMSE scored using Collier2018
[INFO][15][confront][SurfaceDownwardSWRadiation/GEWEX.SRB][ENSEMBLE-CMIP6] Success
[INFO][15][AnalysisMeanStateSites][rsds] Bias scored using Collier2018
[INFO][15][AnalysisMeanStateSites][rsds] RMSE scored using Collier2018
[INFO][15][confront][SurfaceDownwardSWRadiation/WRMC.BSRN][CARDAMOM] Success
[INFO][15][AnalysisMeanStateSites][rsds] Bias scored using Collier2018
[INFO][15][AnalysisMeanStateSites][rsds] RMSE scored using Collier2018
[INFO][15][confront][SurfaceDownwardSWRadiation/WRMC.BSRN][ENSEMBLE-CMIP5] Success
[INFO][15][AnalysisMeanStateSites][rsds] Bias scored using Collier2018
[INFO][15][AnalysisMeanStateSites][rsds] RMSE scored using Collier2018
[INFO][15][confront][SurfaceDownwardSWRadiation/WRMC.BSRN][ENSEMBLE-CMIP6] Success
[INFO][13][AnalysisMeanStateSpace][pr] RMSE scored using Collier2018
[INFO][13][confront][Precipitation/GPCCv2018][ENSEMBLE-CMIP5] Success
[DEBUG][15][extractTimeSeries][CARDAMOM] Could not find [rlds] in the model results
[DEBUG][15][WorkConfront][SurfaceDownwardLWRadiation/CERESed4.2][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][14][extractTimeSeries][CARDAMOM] Could not find [rhums,hurs] in the model results
[DEBUG][14][WorkConfront][SurfaceRelativeHumidity/CRU4.02][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][15][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [rlds] in the model results
[DEBUG][15][WorkConfront][SurfaceDownwardLWRadiation/CERESed4.2][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][15][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [rlds] in the model results
[DEBUG][15][WorkConfront][SurfaceDownwardLWRadiation/CERESed4.2][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][15][extractTimeSeries][CARDAMOM] Could not find [rlds] in the model results
[DEBUG][15][WorkConfront][SurfaceDownwardLWRadiation/FLUXNET2015][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][15][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [rlds] in the model results
[DEBUG][15][WorkConfront][SurfaceDownwardLWRadiation/FLUXNET2015][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][15][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [rlds] in the model results
[DEBUG][15][WorkConfront][SurfaceDownwardLWRadiation/FLUXNET2015][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][14][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [rhums,hurs] in the model results
[DEBUG][14][WorkConfront][SurfaceRelativeHumidity/CRU4.02][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][15][extractTimeSeries][CARDAMOM] Could not find [rlds] in the model results
[DEBUG][15][WorkConfront][SurfaceDownwardLWRadiation/GEWEX.SRB][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][14][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [rhums,hurs] in the model results
[DEBUG][14][WorkConfront][SurfaceRelativeHumidity/CRU4.02][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][13][MakeComparable][Precipitation/GPCCv2018][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [444 360 720] after: [444 273 710]
[INFO][13][MakeComparable][Precipitation/GPCCv2018][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [420  96 144] after: [420  73 142]
[INFO][14][MakeComparable][SurfaceDownwardSWRadiation/CERESed4.2][CARDAMOM] Spatial data was clipped from the reference:  before: [280 180 360] after: [280 137 355]
[DEBUG][15][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [rlds] in the model results
[DEBUG][15][WorkConfront][SurfaceDownwardLWRadiation/GEWEX.SRB][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][15][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [rlds] in the model results
[DEBUG][15][WorkConfront][SurfaceDownwardLWRadiation/GEWEX.SRB][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][15][extractTimeSeries][CARDAMOM] Could not find [rlds] in the model results
[DEBUG][15][WorkConfront][SurfaceDownwardLWRadiation/WRMC.BSRN][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][15][extractTimeSeries][ENSEMBLE-CMIP5] Could not find [rlds] in the model results
[DEBUG][15][WorkConfront][SurfaceDownwardLWRadiation/WRMC.BSRN][ENSEMBLE-CMIP5]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][15][extractTimeSeries][ENSEMBLE-CMIP6] Could not find [rlds] in the model results
[DEBUG][15][WorkConfront][SurfaceDownwardLWRadiation/WRMC.BSRN][ENSEMBLE-CMIP6]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][14][AnalysisMeanStateSpace][rsds] Bias scored using Collier2018
[INFO][14][AnalysisMeanStateSpace][rsds] RMSE scored using Collier2018
[INFO][14][confront][SurfaceDownwardSWRadiation/CERESed4.2][CARDAMOM] Success
[INFO][14][MakeComparable][SurfaceDownwardSWRadiation/CERESed4.2][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [280 180 360] after: [280 137 355]
[INFO][14][MakeComparable][SurfaceDownwardSWRadiation/CERESed4.2][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [ 70  96 144] after: [ 70  73 142]
[INFO][2][AnalysisMeanStateSpace][lai] Bias scored using Collier2018
[INFO][14][AnalysisMeanStateSpace][rsds] Bias scored using Collier2018
[INFO][14][AnalysisMeanStateSpace][rsds] RMSE scored using Collier2018
[INFO][14][confront][SurfaceDownwardSWRadiation/CERESed4.2][ENSEMBLE-CMIP5] Success
[INFO][14][MakeComparable][SurfaceDownwardSWRadiation/CERESed4.2][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [280 180 360] after: [280 137 355]
[INFO][14][MakeComparable][SurfaceDownwardSWRadiation/CERESed4.2][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [178  96 144] after: [178  73 142]
[INFO][2][AnalysisMeanStateSpace][lai] RMSE scored using Collier2018
[INFO][2][confront][LeafAreaIndex/AVH15C1][ENSEMBLE-CMIP6] Success
[INFO][2][MakeComparable][LeafAreaIndex/MODIS][CARDAMOM] Spatial data was clipped from the reference:  before: [ 72 360 720] after: [ 72 273 710]
[INFO][14][AnalysisMeanStateSpace][rsds] Bias scored using Collier2018
[INFO][14][AnalysisMeanStateSpace][rsds] RMSE scored using Collier2018
[INFO][14][confront][SurfaceDownwardSWRadiation/CERESed4.2][ENSEMBLE-CMIP6] Success
[INFO][14][AnalysisMeanStateSites][rsds] Bias scored using Collier2018
[INFO][14][AnalysisMeanStateSites][rsds] RMSE scored using Collier2018
[INFO][14][confront][SurfaceDownwardSWRadiation/FLUXNET2015][CARDAMOM] Success
[INFO][14][AnalysisMeanStateSites][rsds] Bias scored using Collier2018
[INFO][14][AnalysisMeanStateSites][rsds] RMSE scored using Collier2018
[INFO][14][confront][SurfaceDownwardSWRadiation/FLUXNET2015][ENSEMBLE-CMIP5] Success
[INFO][14][AnalysisMeanStateSites][rsds] Bias scored using Collier2018
[INFO][14][AnalysisMeanStateSites][rsds] RMSE scored using Collier2018
[INFO][14][confront][SurfaceDownwardSWRadiation/FLUXNET2015][ENSEMBLE-CMIP6] Success
[INFO][14][MakeComparable][SurfaceDownwardSWRadiation/GEWEX.SRB][CARDAMOM] Spatial data was clipped from the reference:  before: [288 360 720] after: [288 273 710]
[INFO][2][AnalysisMeanStateSpace][lai] Bias scored using Collier2018
[INFO][2][AnalysisMeanStateSpace][lai] RMSE scored using Collier2018
[INFO][2][confront][LeafAreaIndex/MODIS][CARDAMOM] Success
[INFO][2][MakeComparable][LeafAreaIndex/MODIS][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [ 72 360 720] after: [ 72 273 710]
[INFO][2][MakeComparable][LeafAreaIndex/MODIS][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [ 72  96 144] after: [ 72  73 142]
[INFO][13][AnalysisMeanStateSpace][pr] Bias scored using Collier2018
[INFO][14][AnalysisMeanStateSpace][rsds] Bias scored using Collier2018
[INFO][13][AnalysisMeanStateSpace][pr] RMSE scored using Collier2018
[INFO][13][confront][Precipitation/GPCCv2018][ENSEMBLE-CMIP6] Success
[INFO][14][AnalysisMeanStateSpace][rsds] RMSE scored using Collier2018
[INFO][14][confront][SurfaceDownwardSWRadiation/GEWEX.SRB][CARDAMOM] Success
[INFO][13][MakeComparable][Precipitation/GPCPv2.3][CARDAMOM] Spatial data was clipped from the reference:  before: [480 360 720] after: [480 273 710]
[INFO][2][AnalysisMeanStateSpace][lai] Bias scored using Collier2018
[INFO][2][AnalysisMeanStateSpace][lai] RMSE scored using Collier2018
[INFO][2][confront][LeafAreaIndex/MODIS][ENSEMBLE-CMIP5] Success
[INFO][2][MakeComparable][LeafAreaIndex/MODIS][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [ 72 360 720] after: [ 72 273 710]
[INFO][2][MakeComparable][LeafAreaIndex/MODIS][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [ 72  96 144] after: [ 72  73 142]
[INFO][13][AnalysisMeanStateSpace][pr] Bias scored using Collier2018
[INFO][13][AnalysisMeanStateSpace][pr] RMSE scored using Collier2018
[INFO][13][confront][Precipitation/GPCPv2.3][CARDAMOM] Success
[INFO][13][MakeComparable][Precipitation/GPCPv2.3][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [480 360 720] after: [480 273 710]
[INFO][13][MakeComparable][Precipitation/GPCPv2.3][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [324  96 144] after: [324  73 142]
[INFO][2][AnalysisMeanStateSpace][lai] Bias scored using Collier2018
[INFO][2][AnalysisMeanStateSpace][lai] RMSE scored using Collier2018
[INFO][2][confront][LeafAreaIndex/MODIS][ENSEMBLE-CMIP6] Success
[DEBUG][2][extractTimeSeries][CARDAMOM] Could not find [nbp] in the model results
[DEBUG][2][WorkConfront][GlobalNetEcosystemCarbonBalance/GCP][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ConfNBP.py", line 98, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ConfNBP.py", line 65, in stageData
    mod = m.extractTimeSeries(self.variable, alt_vars=self.alternate_vars)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][2][extractTimeSeries][CARDAMOM] Could not find [nbp] in the model results
[DEBUG][2][WorkConfront][GlobalNetEcosystemCarbonBalance/Hoffman][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ConfNBP.py", line 98, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ConfNBP.py", line 65, in stageData
    mod = m.extractTimeSeries(self.variable, alt_vars=self.alternate_vars)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][2][extractTimeSeries][CARDAMOM] Could not find [ra] in the model results
[DEBUG][2][WorkConfront][NetEcosystemExchange/FLUXNET2015][CARDAMOM]
Traceback (most recent call last):
  File "/home/renatob/miniconda3/envs/ilamb/bin/ilamb-run", line 563, in WorkConfront
    c.confront(m)
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 433, in confront
    obs, mod = self.stageData(m)
               ^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/Confrontation.py", line 371, in stageData
    mod = m.extractTimeSeries(
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 409, in extractTimeSeries
    v = self.derivedVariable(
        ^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 495, in derivedVariable
    Vs[arg] = self.extractTimeSeries(
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/renatob/miniconda3/envs/ilamb/lib/python3.11/site-packages/ILAMB/ModelResult.py", line 433, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[INFO][13][AnalysisMeanStateSpace][pr] Bias scored using Collier2018
[INFO][13][AnalysisMeanStateSpace][pr] RMSE scored using Collier2018
[INFO][13][confront][Precipitation/GPCPv2.3][ENSEMBLE-CMIP5] Success
[INFO][1][AnalysisMeanStateSites][gpp] Bias scored using Collier2018
[INFO][1][AnalysisMeanStateSites][gpp] RMSE scored using Collier2018
[INFO][1][confront][GrossPrimaryProductivity/FLUXNET2015][CARDAMOM] Success
[INFO][1][AnalysisMeanStateSites][gpp] Bias scored using Collier2018
[INFO][1][AnalysisMeanStateSites][gpp] RMSE scored using Collier2018
[INFO][1][confront][GrossPrimaryProductivity/FLUXNET2015][ENSEMBLE-CMIP5] Success
[INFO][1][AnalysisMeanStateSites][gpp] Bias scored using Collier2018
[INFO][1][AnalysisMeanStateSites][gpp] RMSE scored using Collier2018
[INFO][1][confront][GrossPrimaryProductivity/FLUXNET2015][ENSEMBLE-CMIP6] Success
[INFO][1][MakeComparable][GrossPrimaryProductivity/FLUXCOM][CARDAMOM] Spatial data was clipped from the reference:  before: [408 360 720] after: [408 273 710]
[INFO][1][AnalysisMeanStateSpace][gpp] Bias scored using Collier2018
[INFO][1][AnalysisMeanStateSpace][gpp] RMSE scored using Collier2018
[INFO][1][confront][GrossPrimaryProductivity/FLUXCOM][CARDAMOM] Success
[INFO][1][MakeComparable][GrossPrimaryProductivity/FLUXCOM][ENSEMBLE-CMIP5] Spatial data was clipped from the reference:  before: [408 360 720] after: [408 273 710]
[INFO][1][MakeComparable][GrossPrimaryProductivity/FLUXCOM][ENSEMBLE-CMIP5] Spatial data was clipped from the comparison:  before: [312  96 144] after: [312  73 142]
[INFO][1][AnalysisMeanStateSpace][gpp] Bias scored using Collier2018
[INFO][1][AnalysisMeanStateSpace][gpp] RMSE scored using Collier2018
[INFO][1][confront][GrossPrimaryProductivity/FLUXCOM][ENSEMBLE-CMIP5] Success
[INFO][1][MakeComparable][GrossPrimaryProductivity/FLUXCOM][ENSEMBLE-CMIP6] Spatial data was clipped from the reference:  before: [408 360 720] after: [408 273 710]
[INFO][1][MakeComparable][GrossPrimaryProductivity/FLUXCOM][ENSEMBLE-CMIP6] Spatial data was clipped from the comparison:  before: [408  96 144] after: [408  73 142]
[INFO][1][AnalysisMeanStateSpace][gpp] Bias scored using Collier2018
[INFO][1][AnalysisMeanStateSpace][gpp] RMSE scored using Collier2018
[INFO][1][confront][GrossPrimaryProductivity/FLUXCOM][ENSEMBLE-CMIP6] Success
[INFO][1][MakeComparable][GrossPrimaryProductivity/WECANN][CARDAMOM] Spatial data was clipped from the reference:  before: [108 180 360] after: [108 137 355]
[INFO][1][AnalysisMeanStateSpace][gpp] Bias scored using Collier2018
[INFO][1][AnalysisMeanStateSpace][gpp] RMSE scored using Collier2018
[INFO][1][confront][GrossPrimaryProductivity/WECANN][CARDAMOM] Success
